tests/datavolume_test.go

```diff
@@ -811,6 +813,50 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		})
 	})
 
+	Describe("Registry import with missing configmap", func() {
+		const cmName = "cert-registry-cm"
+
+		It("Import POD should remain pending until CM exists", func() {
+			var pvc *corev1.PersistentVolumeClaim
+
+			dataVolumeDef := utils.NewDataVolumeWithRegistryImport("missing-cm-registry-dv", "1Gi", tinyCoreIsoRegistryURL)
+			dataVolumeDef.Spec.Source.Registry.CertConfigMap = cmName
+			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolumeDef)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("verifying pvc was created")
+			Eventually(func() bool {
+				// TODO: fix this to use the mechanism to find the correct PVC once we decouple the DV and PVC names
+				pvc, _ = f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(dataVolume.Name, metav1.GetOptions{})
+				return pvc != nil && pvc.Name != ""
+			}, timeout, pollingInterval).Should(BeTrue())
+
+			By("Verifying the POD remains pending for 30 seconds")
+			podName := naming.GetResourceName(common.ImporterPodName, pvc.Name)
+			for i := 0; i < 30; i++ {
```

Comment:
Why not use `Consistently` here?

Reply:
Because I didn't know Consistently existed. I will modify as it much clearer intend.

Reply:
That's a fair reason :wink: :+1: